plane/src/types/mod.rs

```diff
@@ -70,6 +70,53 @@ impl FromStr for ClusterName {
     }
 }
 
+#[derive(
+    Default, Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash, valuable::Valuable,
+)]
+pub struct DronePoolName(String);
+
+impl DronePoolName {
+    pub fn is_default(&self) -> bool {
+        self == &Self::default()
```

Comment:
I was curious if this would involve an allocation since it creates a new string under the hood, but it turns out it does not! https://doc.rust-lang.org/beta/alloc/string/struct.String.html#method.new

(Not that I encourage that sort of optimization anyway, I was just curious)